### DISCUSSION TITLE
Output Log

### DISCUSSION THREAD

**Discussion by radunss** [teler-sh/teler#330] [Q&A] [answered]
I've done most of the configuration but it seems ike it's missing something. My teler doesn't have an option to set and ouput log. I've seen the documentation, it says -o is used but when I typed it it says not defined. Maybe I misread something and it's completely my fault but does the later versions not have that feature? Sorry if this is a stupid question

![help](https://github.com/kitabisa/teler/assets/78066482/21acfb2d-04a2-4be7-a648-f4e63bb537e2)

**Comment by dwisiswant0** [comment, marked as answer]
No need to apologize! It's not a stupid question at all. I understand that you're looking for a way to set the output log in Teler. In the latest versions of teler, the option to save logs is indeed configured through the configuration file, rather than using a command-line option like `-o`.

To set the output log in teler, you need to modify the configuration file directly. You can find an example of the configuration file on the Teler GitHub repository. Specifically, you can refer to the `teler.example.yaml` file, which includes an example of how to configure the output log.

Here is the link to the specific lines in the `teler.example.yaml` file where the output log is configured: [teler.example.yaml#L64-L68](https://github.com/kitabisa/teler/blob/v2/teler.example.yaml#L64-L68).

In that section of the file, you can specify the path where you want the logs to be saved, the JSON format, the enable option, and other relevant options. Once you've made the necessary modifications to the configuration file, save it and restart Teler for the changes to take effect.

I hope this helps! Let me know if you have any further questions.

**Reply by radunss** [reply]
Ah I see, I missed that. Thank you for answering my question.  I'll try it first and get back to you. Thank you again !

**Reply by radunss** [reply]
Thank you!!! It worked flawlessly.
